### ISSUE TITLE
Getting static feedback after playing clips

### ISSUE THREAD

**Issue by Nightwielder** [SilentSys/SLAM#32]
Since the release of version 1.3.1, I am no longer able to play any imported audio clips (mp3s tested) using my bind key (F5 in this case)... just get a persistent hissing noise and I can hear out of game audio (keyclicks, etc.). I'm unsure of what changed that appears to have broken this.

I'm running Windows 10 Pro 64-bit (latest build) and .NET 4.6.

Upon launching CSGO the first time after firing up this version, I received the following message in a window: "Only part of a ReadProcessMemory or WriteProcessMemory request was completed.", followed by the rest of this info. logged in the errorlog.txt file contained within the running program directory:

System.ComponentModel.Win32Exception (0x80004005): Only part of a ReadProcessMemory or WriteProcessMemory request was completed
   at System.Diagnostics.NtProcessManager.GetModuleInfos(Int32 processId, Boolean firstModuleOnly)
   at System.Diagnostics.NtProcessManager.GetFirstModuleInfo(Int32 processId)
   at System.Diagnostics.Process.get_MainModule()
   at SLAM.Form1.PollRelayWorker_DoWork(Object sender, DoWorkEventArgs e)

...However, launching the application and game a second time after a complete exit did not produce this result, but I am unable to play any audio files after selecting them in the developer console. I did run the "exec slam" command, and the "la" option brings up my playlist selection, but after loading any of these tracks, the output is just static. Please advise on a fix!

Thanks for all your work on this project.

**Comment by charmanjoe** [comment]
Did you try binding the slam_play command manually?  I found that after choosing a song the key that is normally bound to slam_play is bound to the song number.  Use "bind x 'slam_play'" after choosing the song and see if it works.

**Comment by Nightwielder** [comment]
@charmanjoe -- thanks for your suggestion... though unfortunately this method did not work for me, unless I'm not understanding you correctly. I started SLAM, went into the game, ran "exec slam", then "la" selecting a track from my list, then I used the manual bind for "slam_play" for my play key in the app. I hit my play key, and I just get static once again.

I'd like to point out a key difference between this version and the last I used -- 1.2.2 -- which allowed you to manually right-click a track from your list in the app and choose to load it on-demand. I had to do this manually for each and every audio clip that I wanted hear in-game using my play key. Without following this process, I was unable to actually 'load' any tracks using the "la" listing and corresponding track number. It will indicate the track is loaded, and that's expected behavior, but when you go to actually play it -- doesn't work, producing the same static feedback in the latest version. Additionally, loading a track using "la" + number doesn't update the app out of game, which I believe is the intention... and the app says the loaded status for the given track is false. In the latest version, there is no option whatsoever to 'load' any tracks on-demand in this way, so clearly something has changed here.
